### DISCUSSION TITLE
2.3.3 Неправильно определил архитектуру

### DISCUSSION THREAD

**Discussion by 777Evgeny777** [Anonym-tsk/nfqws-keenetic#80] [Делимся опытом]
OPENWRT 19.07.10 Роутер микротик RB750Upr2 , определил как mipsel, а должен был как mips32r1-msb. Соответсвенно nfqws в процессах нет.
Я так думаю. Вручную запихал nfqws-mips (переименовал естественно и права дал), заработало.

**Comment by Anonym-tsk** [comment]
Покажите вывод
```
uname -a
cat /proc/cpuinfo
cat /etc/opkg/distfeeds.conf
```

**Reply by 777Evgeny777** [reply]
root@OpenWrt:~# uname -a
Linux OpenWrt 4.14.275 #0 Sat Apr 16 13:13:32 2022 mips GNU/Linux
root@OpenWrt:~# cat /proc/cpuinfo
system type             : Qualcomm Atheros QCA9533 ver 2 rev 0
machine                 : MikroTik RouterBOARD 750UP r2
processor               : 0
cpu model               : MIPS 24Kc V7.4
BogoMIPS                : 366.18
wait instruction        : yes
microsecond timers      : yes
tlb_entries             : 16
extra interrupt vector  : yes
hardware watchpoint     : yes, count: 4, address/irw mask: [0x0ffc, 0x0ffc, 0x0ffb, 0x0ffb]
isa                     : mips1 mips2 mips32r1 mips32r2
ASEs implemented        : mips16
Options implemented     : tlb 4kex 4k_cache prefetch mcheck ejtag llsc dc_aliases perf_cntr_intr_bit nan_legacy nan_2008 perf
shadow register sets    : 1
kscratch registers      : 0
package                 : 0
core                    : 0
VCED exceptions         : not available
VCEI exceptions         : not available

root@OpenWrt:~# cat /etc/opkg/distfeeds.conf
src/gz openwrt_core http://downloads.openwrt.org/releases/19.07.10/targets/ar71xx/mikrotik/packages
src/gz openwrt_kmods http://downloads.openwrt.org/releases/19.07.10/targets/ar71xx/mikrotik/kmods/4.14.275-1-45ac9e65a50ef93760062d2f90a31388
src/gz openwrt_base http://downloads.openwrt.org/releases/19.07.10/packages/mips_24kc/base
src/gz openwrt_freifunk http://downloads.openwrt.org/releases/19.07.10/packages/mips_24kc/freifunk
src/gz openwrt_luci http://downloads.openwrt.org/releases/19.07.10/packages/mips_24kc/luci
src/gz openwrt_packages http://downloads.openwrt.org/releases/19.07.10/packages/mips_24kc/packages
src/gz openwrt_routing http://downloads.openwrt.org/releases/19.07.10/packages/mips_24kc/routing
src/gz openwrt_telephony http://downloads.openwrt.org/releases/19.07.10/packages/mips_24kc/telephony

**Reply by 777Evgeny777** [reply]
Да не замарачивайтесь, это уже в каком-то начальном пакете встречалось. Стимулирует головой думать. Решается то легко проблема.

**Comment by Anonym-tsk** [comment]
В следующем обновлении будет исправлено
